code/game/objects/items/weapons/clown_items.dm

```diff
@@ -59,6 +60,11 @@
 	if(honk())
 		add_fingerprint(user)
 
+/obj/item/weapon/bikehorn/Crossed(var/mob/living/AM)
```

Comment:
You're typing the arg as `/mob/living` but not checking that it actually is that type, so it still has the problem of honking when anything at all moves over it

Reply:
I copied it from glass shards lol

Reply:
Yes, this line, but not the rest of it

Reply:
Added that check, done